elements/lisk-p2p/src/constants.ts

```diff
@@ -55,6 +55,8 @@ export const DEFAULT_PRODUCTIVITY = {
 // Peer inbound constants
 export const DEFAULT_PING_INTERVAL_MAX = 60000;
 export const DEFAULT_PING_INTERVAL_MIN = 20000;
+export const SOCKET_PING_MESSAGE = '#1';
```

Comment:
Where is this coming from? It looks too specific. I would suggest to have a regEx capturing similar cases.